lib/hashie/extensions/persistable.rb

```diff
@@ -0,0 +1,38 @@
+  module Hashie
+    module Extensions
+      module Persistable
+        CANNOT_INCLUDE = 'Peristable can only be used on classes with Hashie::Extensions::HashInitializer'
```

Comment:
I think just this:

```ruby
class MissingHashInitializer < StandardError; end
```

I would just not include the lengthy explanation, if we do it should be a localizable thing, etcetera.